packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -310,6 +310,7 @@ export type GraphiQLState = {
   operations?: OperationDefinitionNode[];
   documentAST?: DocumentNode;
   maxHistoryLength: number;
+  darkMode: boolean;
```

Comment:
do we want to provide a `defaultTheme`: "light" | "dark" prop so that implementers can default to dark mode for their users?